src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Controller/ConfigurationControllerTest.php

```diff
@@ -38,7 +38,7 @@ public function setUp()
         $this->loadFixtures([]);
 
         $this->client = static::createClient();
-        $this->password = $this->client->getKernel()->getContainer()->getParameter('management_password');
+        $this->password = 'management_password_test';
```

Comment:
Please fix the issue at hand under boyscouting, please ensure the `InvalidCommandExecptionListener` is prioritized over the generic `ExceptionListener` (see prioroties listed [here][1])

[1]: http://symfony.com/doc/2.7/reference/events.html#kernel-exception